pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -985,10 +965,23 @@ func (b *MasqueradeBindMechanism) createBridge() error {
 }
 
 func (b *MasqueradeBindMechanism) createNatRules(protocol iptables.Protocol) error {
-	if Handler.HasNatIptables(protocol) {
+	err := Handler.ConfigureIpForwarding(protocol)
+	if err != nil {
+		log.Log.Reason(err).Errorf("failed to configure ip forwarding")
+		return err
+	}
+
+	ipString := "ipv4"
```

Comment:
The variable part here, imho, is just 4 / 6 . The `ipv` portion would be part of the template.

Reply:
done